modules/viewing-security-information-in-red-hat-developer-hub.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * configuration/configuring-and-integrating-the-rhacs-plugin-with-red-hat-developer-hub.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-security-information-in-red-hat-developer-hub_{context}"]
+= Viewing security information in Red Hat Developer Hub
+
+By configuring and integrating the {rh-rhacs-first} plugin with {rh-rhdh-first}, you can access vulnerability data, assess risks, and take proactive security actions without leaving the {rh-rhdh} environment.
+
+.Prerequisites
+
+* You have enabled the {product-title-short} plugin installation in {rh-rhdh}.
++
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{rhdh-latest-version}/html/installing_and_viewing_plugins_in_red_hat_developer_hub/rhdh-installing-rhdh-plugins_title-plugins-rhdh-about#con-install-dynamic-plugin-helm_rhdh-installing-rhdh-plugins[Installing dynamic plugins using the Helm chart] ({rh-rhdh} documentation).
+
+.Procedure
+
+. Create an `app-config.yaml` file that contains the `proxy` and `acs` stanzas by using the following content:
++
+[source,yaml]
+----
+# ...
+proxy:
+  endpoints:
+    /acs:
+      target: ${ACS_API_URL}
+      headers:
+        authorization: Bearer ${ACS_API_KEY}
+acs:
+  acsUrl: ${ACS_API_URL}
+# ...
+----
+
+. To enable the {product-title-short} plugin, perform the following steps:
+.. Navigate to the dynamic plugins configuration file in your {rh-rhdh} setup, for example, `app-config.yaml`.
```

Comment:
Sorry, just noticed this! The dynamic plugins configurations file will be different than the `app-config.yaml`. The dynamic plugins config is apart of RHDH and is an additional step for configuring the plugin with RHDH. The app-config.yaml is used for both Backstage and RHDH.

I would just remove this from portion of this sentence: ", for example, `app-config.yaml`."